src/main/scala/nl/knaw/dans/easy/ps/DDM.scala

```diff
@@ -113,12 +113,35 @@ object DDM {
 
   def createComposedCreator(dictionary: Dictionary, authorFields: Iterable[(MdKey, String)]) = {
     <dcx-dai:creatorDetails>
-      <dcx-dai:author>
-        { authorFields.map(composedEntry(dictionary)) }
-      </dcx-dai:author>
+      {
```

Comment:
What happens if you mix "person" creators and "organization-only" creators? Maybe add a unit-test to show that that is handled correctly?